perennialAutodidact/memory-snap
# 22 backend endpoint for returning pexels image array

## Description of changes
- Created an endpoint for which calls the Pexels API and returns an array of photos
- Added configuration for Jest, ESLint, and Prettier
- Added Windows Git configuration for line endings
- Created Pexels mock and wrote basic tests for the `/` endpoint

## Test steps
- In the root directory, run `yarn install`
- Navigate into the `packages/backend/` directory
- Run `yarn test`
- All tests should pass

I didn't hook up the client to the API, so there isn't much to test other than the two unit tests I wrote for the endpoint. All backend tests should pass.
